src/sections/Company/About/index.js

```diff
@@ -14,9 +14,37 @@ import CNCFstackedlogo from "./images/cncf-stacked-color.svg";
 import CNCFLightstackedlogo from "./images/cncf-light-stacked-color.svg";
 
 import { Link } from "gatsby";
+import { graphql, useStaticQuery } from "gatsby";
+import { getImage } from "gatsby-plugin-image";
+import { BgImage } from "gbimage-bridge";
+
 const Community_meetup = "./images/Community_meetup.png";
-const Layer5Projects = "./images/layer5-projects.png";
+// const Layer5Projects = "./images/layer5-projects.svg";
+
+
+
 const About = ({ theme }) => {
+
+  // Lee Calcote learned this trick from Maintainer Nikhil Ladha
+  const { webpImage } = useStaticQuery(
+    graphql`
+      query {
+        webpImage: file(
+          relativePath: { eq: "./images/layer5-projects.svg"" }
```

Comment:
If we are using the gatsby-plugin, then let's use the PNG image itself.
SVG doesn't work correctly with the plugin.

Reply:
And, since it is a relative path, we don't need to provide the folder name if it's under the root (here `images`) directory.
i.e, we can simply use `layer5-projects.png`

Reply:
Also, we have a extra double quote at the end of the relativePath